voc/function.go

```diff
@@ -31,8 +31,9 @@ var FunctionType = []string{"Function", "Compute", "Resource"}
 
 type Function struct {
 	*Compute
-	HttpEndpoint        *HttpEndpoint `json:"httpEndpoint"`
-	RuntimeVersion      string        `json:"runtimeVersion"`
-	RuntimeLanguage     string        `json:"runtimeLanguage"`
-	PublicNetworkAccess bool          `json:"publicNetworkAccess"`
+	HttpEndpoint    *HttpEndpoint `json:"httpEndpoint"`
```

Comment:
Speaking of http endpoint: It should include a public access field, shouldn't it?

Reply:
I think yes, but I think that is something for our developer meeting.